README.md

```diff
@@ -22,7 +22,7 @@ If Vim doesn't recognize the file type, you could add this to your `.vimrc`:
 autocmd BufRead,BufNewFile pattern_matched_your_file(s) set filetype=nginx
 ```
 
-For example, `autocmd BufRead,BufNewFile nginx_*.conf.tpl set filetype=nginx`.
+For example, `autocmd BufRead,BufNewFile nginx_*.conf.tpl set filetype=nginx` or `autocmd BufRead,BufNewFile * if match(getline(1),"nginx") >=0 | set filetype=nginx | endif`
```

Comment:
`match(getline(1),"nginx")` means a file whose first line contains `nginx`?

Reply:
> `match(getline(1),"nginx")` means a file whose first line contains `nginx`?

yes, it means the first line contains `nginx`.  
 under MacOS , if i want to it works need i  `:set ft=nginx` in last line. because it usually autoloads the `nginx.vim` under $VIM , which makes it impossible to load the openResty-vim plugin.